### PR TITLE
Lrapetti/update trevis

### PR DESCRIPTION
**Pull request by lrapetti**
test disable test stage

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,19 +13,38 @@ stages:
...
+    - boost
+    - tinyxml
+    - swig
+    - qt5
+    - gsl
+    - pkg-config
+    - jpeg
+    - sqlite
+    - tinyxml
+    - gazebo9
```
Comment (traversaro) [inline .travis.yml]: I suspect including `gazebo9` here will create you problems, as `gazebo9` is installed by the custom tap `osrf/simulation`. As for sure `gazebo9` will never be installed by default in the Travis image, I think it is ok to install it via `brew install gazebo9`, **after** the `brew tap osrf/simulation` (or you can also install directly with `brew install osrf/simulation/gazebo9` .

Comment (lrapetti) [reply .travis.yml]: yess, I see. I was just testing the addons.

> you can also install directly with `brew install osrf/simulation/gazebo9`

The homebrew addons should support also `tap`, so we may be able to do it directly there.
Something like:
```
addons:
  homebrew:
    taps:
        - osrf/simulation
    packages:
        - gazebo9
```

Comment (traversaro) [reply .travis.yml]: Great!

**Comment by lrapetti** [comment]
I teste Trevis using the `addons` trying both to insert them into `stage: osx`:
```
jobs:
  include:
    # ---------
    # STAGE OSX
    # ---------
    - &osx_template
      stage: osx
      os: osx
      osx_image: xcode9.4
      before_install: skip
      install: *osx_install
      before_script: skip
      script: *osx_script_xcode
      after_failure: skip
      after_success: skip
      after_script: skip
      env:
        TRAVIS_CMAKE_GENERATOR="Xcode"
        TRAVIS_BUILD_TYPE="Debug"
      addons:
         homebrew:
         ....
```
or defining them before like in https://github.com/robotology/yarp/blob/master/.travis.yml#L49, but I don't see them being used.

Furthermore, the build is also failing but the check return passed (see https://travis-ci.com/lrapetti/robotology-superbuild/builds/95231404#L403)

**Comment by lrapetti** [comment]
cc @diegoferigo

**Comment by traversaro** [comment]
An alternative is to define a `Brewfile` as discussed in https://github.com/robotology/robotology-superbuild/issues/141 .

**Comment by lrapetti** [comment]
> An alternative is to define a `Brewfile` as discussed in [robotology#141](https://github.com/robotology/robotology-superbuild/issues/141).

But it is not solving the problem since it is equivalent to running `brew install` and it is failing as before (see https://travis-ci.com/lrapetti/robotology-superbuild/builds/95241209). 
I don't think that even `homebrew addons` take care of the case in which a `package` is already installed.

**Comment by traversaro** [comment]
I guess that `brew bundle` does not complain if the package is already installed, but it complains instead if the package is already installed not by homebrew, if I understand correctly what is written in https://github.com/Homebrew/homebrew-bundle/issues/313 .

**Comment by traversaro** [comment]
If nothing else works, we can just remove the offending formula from `brew install` . We will be vulnerable to new changes  in the Travis macOS image, but that is the minor evil probable.

**Comment by lrapetti** [comment]
> If nothing else works, we can just remove the offending formula from `brew install` . We will be vulnerable to new changes in the Travis macOS image, but that is the minor evil probable.

Yes, I just tested without `sqlite` and `boost` and it is compiled correctly.

**Comment by diegoferigo** [comment]
To avoid the warning on the unsupported services on macos, you can do this:

```yaml
jobs:
  include:
    # ===================
    # STAGE: test (linux)
    # ===================
    - stage: test
      if: env(DUMMYVAR) is present
    # ---------
    # STAGE OSX
    # ---------
    - &osx_template
      stage: osx
      os: osx
      services:
      othertags: ...
```

**Comment by lrapetti** [comment]
The support to MacOS in now restored by removing the packages, I think we can do a PR in robotology.

Given that for the moment it was not possible to use the `addons`, we can decide to go through `Brewfile` or leave it as it is.
@traversaro @diegoferigo What do you think?

**Comment by traversaro** [comment]
As you prefer @lrapetti . I would probably go for just removing the installed packages from the relevant `.travis.yml` line, as this is easier and more maintainable rather than go for a separate file.

**Comment by diegoferigo** [comment]
I agree with @traversaro :+1:

**Comment by lrapetti** [comment]
Good old time with Trevis CI 😄  I think we can close it

**Pull request closed without merging**